src/github_bot/mod.rs

```diff
@@ -60,7 +60,27 @@ impl GithubBot {
 			repo = repo_name,
 			sha = sha
 		);
-		self.client.get(url).await
+		let result: Result<github::CombinedStatus> = self.client.get(url).await;
+		result.map(|ctx| {
+			github::CombinedStatus {
+				statuses: ctx
+					.statuses
+					.iter()
+					.filter_map(|s| {
+						// FIXME stopgap hardcoded measure until vanity-service encodes which jobs
+						// can be skipped into the status' description
+						if s.context
+							== "continuous-integration/gitlab-cargo-deny"
+						{
+							return None;
+						}
+
+						Some(s.clone())
+					})
+					.collect(),
+				..ctx
+			}
+		})
```

Comment:
can be simplified a bit
```suggestion
		let mut result: Result<github::CombinedStatus> = self.client.get(url).await;
		// FIXME stopgap hardcoded measure until vanity-service encodes which jobs
		// can be skipped into the status' description
		// https://github.com/paritytech/parity-processbot/issues/242
		if let Ok(combined) = result.as_mut() {
			combined.statuses.retain(|s| {
				s.context != "continuous-integration/gitlab-cargo-deny"
			})
		}
		result
```

Reply:
done